### ISSUE TITLE
Balloon.py index of detected object as output

### ISSUE THREAD

**Issue by darvida** [matterport/Mask_RCNN#980]
Hey im using balloon.py with the balloon dataset. Is there a way to get the (x,y)-index of a detected objects mask ? Right now im using the `python3 balloon.py splash --weights=/path/to/weights/file.h5 --image=<URL or path to file>` command to detect balloons. 
Is there a way in tensorflow or keras to get the detected objects masks index as an output?
